### DISCUSSION TITLE
Link tables between projects [Oct 01]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6618] [Discord Threads]
> 🙋‍♂️ **probstenhias**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Not supported currently.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1146805968605286471)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
